README.md

```diff
@@ -51,3 +51,7 @@ $ dci-pipeline cnf:ansible_inventory=/var/lib/dci/inventories/dallas/8nodes/clus
 ```
 TBD
 ```
+
+### Upgrade validation
+When run during an upgrade, an extra DCI hook can be called to gather information from the CNF behavior.
```

Comment:
- run = running; or maybe you can directly say "When running an upgrade"
- "from the CNF behavior" -> "about the CNF behavior"?

Reply:
You are rigth, about the CNF behavior looks better, but I don't see the pb with the first one.

Reply:
You're right also with my first comment, it's fine as it is right now, I'll close this